From a307120133ea36dcf1013d35dfbed3bad55a6db8 Mon Sep 17 00:00:00 2001
From: Rohit Athavale <rathaval@xilinx.com>
Date: Fri, 23 Feb 2018 11:15:06 -0800
Subject: [PATCH 0305/1852] v4l: xilinx-gamma: Fix compile error in debug mode

commit f7548c47294b9672071854f01b2ce9a88e640e6a from
https://github.com/Xilinx/linux-xlnx.git

Variable rb is not declared in DEBUG mode, but is referenced.
This fixes the dev_err logging.

Signed-off-by: Rohit Athavale <rathaval@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-gamma.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-gamma.c b/drivers/media/platform/xilinx/xilinx-gamma.c
index 47e25fc81517..bc1b0e358942 100644
--- a/drivers/media/platform/xilinx/xilinx-gamma.c
+++ b/drivers/media/platform/xilinx/xilinx-gamma.c
@@ -97,8 +97,7 @@ static inline void xg_write(struct xgamma_dev *xg, u32 reg, u32 data)
 #ifdef DEBUG
 	if (xg_read(xg, reg) != data)
 		dev_err(xg->xvip.dev,
-			"Wrote 0x%x does not match read back 0x%x",
-			data, rb);
+			"Write 0x%x does not match read back", data);
 #endif
 }
 
-- 
2.31.1

